src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -466,6 +478,42 @@ public NativeWindow(INativeWindowProperties settings)
                         break;
                 }
 
+                switch (API)
+                {
+                    case ContextAPI.NoAPI:
```

Comment:
This is more of a nicety and doesn't necessarily need to be actioned, but it'd be nice to have `ContextAPI.Vulkan`, even though it has the same effect. Would be easy, just add an enum member and then change this to:
```cs
case ContextAPI.NoAPI:
case ContextAPI.Vulkan:
```

Reply:
I think that might be confusing, since we don't have any sort of Vulkan support yet.

Reply:
We will do shortly, however. There's a fair chance that I can get it done for 4.0's debut, if not in the next release after.

Reply:
Fair enough, will do.